arch/arm/boot/dts/overlays/edt-ft5406.dtsi

```diff
@@ -0,0 +1,54 @@
+/*
+ * Device Tree overlay for an EDT FT5406 touchscreen
+ *
+ */
+
+/ {
+	compatible = "brcm,bcm2835";
+
+	fragment@10 {
```

Comment:
Starting the shared numbering at 10 is an unusual decision, but it should work now (there was a time when it wouldn't have).

Reply:
This was a question from a while back - was there benefit in having edt-ft5406 distinct from vc4-kms-dsi-7inch. There's nothing stopping running one without the other, so sharing the config in a dtsi was preferred.  The fragment numbers therefore need to be unique, but we could shift the ones in vc4-kms-dsi-7inch instead.

Reply:
> we could shift the ones in vc4-kms-dsi-7inch instead.

That isn't necessary, but it is less surprising.

Reply:
I've added a comment to the top of edt-ft5406.dtsi to explain the odd fragment numbers.